error building with react native 0.79

I'm seeing the error below when building for react native 0.79 with the new arch enabled:

```
.../React-RuntimeApple/React_RuntimeApple.framework/Headers/ReactCommon/RCTHost.h:12:9: fatal error: 'react/runtime/JSRuntimeFactory.h' file not found
   12 | #import <react/runtime/JSRuntimeFactory.h>
```

I'm not sure where to begin debugging, as it seems like some sort of head path configuration issue.

--- a/react-native-ios-utilities.podspec
+++ b/react-native-ios-utilities.podspec
@@ -144,6 +144,8 @@ Pod::Spec.new do |s|
   if use_hermes && reactNativeTargetVersion >= 78
     add_dependency(s, "React-jsinspector", :framework_name => 'jsinspector_modern')
     add_dependency(s, "React-jsinspectortracing", :framework_name => 'jsinspector_moderntracing')
+    # bring in the new C-API runtime headers for React Native 0.79
+    add_dependency(s, "React-jsitooling", :framework_name => 'jsitooling')
   end
 
   s.dependency 'React-Core'

